Controllers/NLog.config

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Please remove NLog.config, NLog.xsd. All appropriate logger configuration sections should be in web.config